src/components/TextField/TextField.scss

```diff
@@ -165,9 +166,13 @@ $stacking-order: (
   position: relative;
   z-index: z-index(contents, $stacking-order);
   flex: 0 0 auto;
-  margin: 0 $addon-horizontal-spacing 0 $backdrop-horizontal-spacing;
-  color: currentColor;
+  margin: 0 $prefix-horizontal-spacing 0 $backdrop-horizontal-spacing;
+  color: color('ink', 'lighter');
```

Comment:
I'm thinking we still would want the `color('ink', 'lightest`') for disabled fields @sarahill?

Reply:
Good catch. Yes, we'll still want `color('ink', 'lightest')` for disabled fields.